test/glgettexenv.c

```diff
@@ -68,8 +68,15 @@ int main(int argc, char *argv[])
     assert(colora[1] == colorb[1]);
     assert(colora[2] == colorb[2]);
     assert(colora[3] == colorb[3]);
+
+    // Check for no compilation errors with the following.
+    if (argc == 1234) {
+      GLint i;
+      glGetTexLevelParameteriv(GL_TEXTURE_2D, 0, GL_TEXTURE_WIDTH, &i);
+    }
```

Comment:
Just taking the address of this function would also be enough to trigger the error.  (i.e. `printf("%p\n", glGetTexLevelParameteriv)` ) although I'm not sure that is more readable.

Reply:
Oh wait, I remember now how I went about testing and fixing this the last time it came up.   We can use `-sINCLUDE_FULL_LIBRARY` for force inclusion of all JS library funcs.

See #19902

Instead of this test I would add a wasm64 or 4gb mode to other.test_full_js_library.  That should catch this as well as any others.

Reply:
Sounds good, done. I also confirmed the new test fails before the fix here.